.travis.yml

```diff
@@ -14,9 +14,11 @@ env:
     - USE_DEB=true
       ROS_DISTRO="indigo"
       ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
+      UPSTREAM_WORKSPACE=file
     - USE_DEB=true
```

Comment:
Same as [the comment above](https://github.com/ros-industrial/staubli_experimental/pull/12/files#r70192123).